junebug/plugins/nginx/plugin.py

```diff
@@ -16,7 +16,7 @@
 
 
 def resource_path(filename):
-    return resource_filename('junebug.plugins.nginx', filename)
+    return path.join(path.dirname(__file__), filename)
```

Comment:
Done. I thought the reason we used `pkg_resources.resource_filename` was because the `__file__` approach wouldn't work in a zip, though.